rcl/test/rcl/test_timer.cpp

```diff
@@ -183,7 +183,7 @@ TEST_F(TestTimerFixture, test_two_timers_ready_before_timeout) {
   rcl_timer_t timer2 = rcl_get_zero_initialized_timer();
 
   ret = rcl_timer_init(
-    &timer, &clock, this->context_ptr, RCL_MS_TO_NS(50), nullptr, rcl_get_default_allocator());
+    &timer, &clock, this->context_ptr, RCL_MS_TO_NS(10), nullptr, rcl_get_default_allocator());
```

Comment:
If this is an important change, can you add a comment so it's clearer why this value was chosen?

Reply:
Dropped a comment in https://github.com/ros2/rcl/pull/680/commits/b19b6f078ac8da5d7f3a0086e9d7dc4b31c17b0e.